code_schemes/rqas/leap_s05/s05e01.json

```diff
@@ -5,10 +5,10 @@
   "Codes": [
     {
       "CodeID": "code-e793c110",
-      "DisplayText": "laziness",
+      "DisplayText": "disempowerment",
```

Comment:
Is this a renaming of the same concept or a new theme?

Reply:
renaming of the same concept.